commit 9f0225894254c4706519c85d6fc06c5382903fef
Author: Wolfgang Pieb <wolfgang_pieb@playstation.sony.com>
Date:   Thu Jul 30 20:36:31 2020 -0700

    Make a test case more flexible on Windows wrt the number of backslashes.

diff --git a/lld/test/ELF/dependency-file.s b/lld/test/ELF/dependency-file.s
index 14048ead494..e7dbf9c7695 100644
--- a/lld/test/ELF/dependency-file.s
+++ b/lld/test/ELF/dependency-file.s
@@ -6,16 +6,16 @@
 # RUN: ld.lld -o %t/foo.exe %t/foo.o %t/"bar baz.o" "%t/#quux$.o" --dependency-file=%t/foo.d
 # RUN: FileCheck --match-full-lines -DFILE=%t %s < %t/foo.d
 
-# CHECK:      [[FILE]]{{/|\\\\}}foo.exe: \
-# CHECK-NEXT:   [[FILE]]{{/|\\\\}}foo.o \
-# CHECK-NEXT:   [[FILE]]{{/|\\\\}}bar\ baz.o \
-# CHECK-NEXT:   [[FILE]]{{/|\\\\}}\#quux$$.o
+# CHECK:      [[FILE]]{{/|(\\)+}}foo.exe: \
+# CHECK-NEXT:   [[FILE]]{{/|(\\)+}}foo.o \
+# CHECK-NEXT:   [[FILE]]{{/|(\\)+}}bar\ baz.o \
+# CHECK-NEXT:   [[FILE]]{{/|(\\)+}}\#quux$$.o
 # CHECK-EMPTY:
-# CHECK-NEXT: [[FILE]]{{/|\\\\}}foo.o:
+# CHECK-NEXT: [[FILE]]{{/|(\\)+}}foo.o:
 # CHECK-EMPTY:
-# CHECK-NEXT: [[FILE]]{{/|\\\\}}bar\ baz.o:
+# CHECK-NEXT: [[FILE]]{{/|(\\)+}}bar\ baz.o:
 # CHECK-EMPTY:
-# CHECK-NEXT: [[FILE]]{{/|\\\\}}\#quux$$.o:
+# CHECK-NEXT: [[FILE]]{{/|(\\)+}}\#quux$$.o:
 
 .global _start
 _start:
